main.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ecs"
+
+	ecs_state "github.com/Wen777/ecs_state"
+	cli "github.com/urfave/cli"
+)
+
+// StartTask on AWS ECS Cluster
+func StartTask(CLUSTER string, TASK string) error {
+
+	// List cluster info
+	sess, _ := session.NewSession(&aws.Config{Region: aws.String("us-west-1")})
+	client := ecs.New(sess)
+
+	state := ecs_state.Initialize(CLUSTER, client)
+	state.RefreshClusterState()
+	state.RefreshContainerInstanceState()
+	state.RefreshTaskState()
+	fmt.Printf("Found Cluster: %+v\n\n", len(state.FindClusterByName(CLUSTER).ContainerInstances))
+
+	// TODO Register task definitions
+	ec2Arr := state.FindClusterByName(CLUSTER).ContainerInstances
```

Comment:
Can you handle the case when cluster name doesn't exist?
Also let's name parameter with camelCase instead of all caps please.